Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -380,6 +380,8 @@
 
     String sessionReferredLink_; // Link which opened this application session if opened by a link click.
 
+    private static String cookieBasedMatchDomain_ = null; // Domain name used for cookie based matching
```

Comment:
So, you're thinking that we don't use a default and that we just ask the dev to specify?